om4labs/diags/stress_curl/example_stress_curl.ipynb

```diff
@@ -0,0 +1,249 @@
+{
+ "cells": [
```

Comment:
I don't think we need to add the example notebook.
They are too heavy because of the plots,...

Reply:
you can make a backup somewhere and `git rm example_stress_curl.ipynb`